odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/utils/JSONSerDeUtils.java

```diff
@@ -43,6 +45,14 @@
         }
     }
 
+    public static <T> T deserializeJson(final File file, final TypeReference<T> tr) {
```

Comment:
Please do not populate main code with the test-designed one. Let's move this method into some test utils class